### PR TITLE
[KIECLOUD-107] Add Prometheus config to EAP-based RHPAM 7 images

### PR DESCRIPTION
**Pull request by errantepiphany**
[KIECLOUD-107] Add Prometheus module to EAP-based RHPAM 7 images
https://issues.jboss.org/browse/KIECLOUD-107

Signed-off-by: David Ward <dward@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[RHPAM-XYZ] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Comment by vblagoje** [comment]
@errantepiphany out of curiosity - why do we need name field set to 'jboss.container.eap.prometheus.config'?

**Comment by errantepiphany** [comment]
> @errantepiphany out of curiosity - why do we need name field set to 'jboss.container.eap.prometheus.config'?

@vblagoje That's not for your stuff. :) That's for if users want to enable EAP prometheus stuff.

**Comment by errantepiphany** [comment]
Labeled as "on hold" as this got pushed to 7.4.

**Comment by errantepiphany** [comment]
@spolti Can you please re-review?

File: kieserver/image.yaml [hunk 1]
```diff
@@ -146,6 +146,9 @@ envs:
     - name: "KIE_SERVER_ROUTER_PORT"
       example: "9000"
       description: "KIE server router port (Used to set the org.kie.server.router system property)"
+    - name: "PROMETHEUS_SERVER_EXT_DISABLED"
```
Comment (spolti) [inline kieserver/image.yaml]: shouldn't we add it for others images as well?

Comment (errantepiphany) [reply kieserver/image.yaml]: @spolti That property is only for the kieserver images. But the AB stuff comes from cct_module and is for all EAP-based images.

**Pull request merged**